src/main/java/calculator/Service.java

```diff
@@ -0,0 +1,37 @@
+package calculator;
+
+public class Service {
+
+    private final static String ADDITION = "+";
+    private final static String SUBTRACTION = "-";
+    private final static String MULTIPLICATION = "*";
+    private final static String DIVISION = "/";
+
+
+    public int start(String input) {
+        Calculator.validateInputNullOrEmpty(input);
+        String[] inputs = input.split(" ");
+        int result = Calculator.validateInputIsNumber(inputs[0]);
+        for (int i = 1; i < inputs.length; i += 2) {
+            int nextInput = Calculator.validateInputIsNumber(inputs[i + 1]);
+            result = calculator(inputs[i], result, nextInput);
```

Comment:
```suggestion
            result = operator.calculate(result, nextInput);
```

Operator를 추상화시켰다면 위와같은 코드가 나올 수 있습니다!